src/dependency.rs

```diff
@@ -77,3 +82,22 @@ pub async fn ensure(base_dir: camino::Utf8PathBuf, deps: Vec<fpm::Dependency>) -
     .await;
     Ok(())
 }
+
+#[derive(serde::Deserialize, Debug, Clone)]
+pub struct DependencyTemp {
+    pub name: String,
+    pub version: Option<String>,
+    pub repo: String,
+    pub notes: Option<String>,
+}
+
+impl DependencyTemp {
+    pub fn to_dependency(&self) -> fpm::Dependency {
```

Comment:
Make this consuming converter, change the name to `into_dependency` and type to `self` and remove all the `to_owned()` calls. Also you have used `.to_owned()` wrong, which is used to convert a reference to an object, here your intention was `.clone()`.